README.md

```diff
@@ -157,6 +160,34 @@ yotiClient.getActivityDetails(token).then((activityDetails) => {
 
 [Example reference](examples/profile/index.js)
 
+
+## Dynamic Policy Generation
+
+Dynamic policy generation is a way of requesting non-static policies for your application
+
+This is useful when you want or need different permutations of attributes from Yoti
+
+E.g:
+ 
+* Request 1: full-name and date-of-birth
+* Request 2: full-name and address
+
+This service will query Yoti Core (via Connect) for a QR Code/link associated with the requested policy.
```

Comment:
It would be good if @KiranBali could review this text - AFAIK `Yoti Core` (and probably `Connect` too) are not terms which clients will be aware of. It might be better if we just state `will query Yoti for a QR Code`.